metabase/resource_collection.go

```diff
@@ -22,14 +22,34 @@ func resourceCollection() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
+			"parent_id": {
+				Description: "Parent collection id",
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+			},
 			"name": {
 				Description: "Collection name",
 				Type:        schema.TypeString,
 				Required:    true,
 			},
+			"default_access": {
+				Description: "Default access for all users",
+				Type:        schema.TypeString,
+				Default:     "none",
+				Optional:    true,
+			},
+			"permissions": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
 			"color": {
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
+				Default:  "#31698A",
```

Comment:
👍🏼 I see that you have yourself addressed this https://github.com/getmoss/terraform-provider-metabase/commit/2edc30babdfc05e82b1de18bb49e5f34d1fe98b3#r81414120